src/main/webapp/main-page.html

```diff
@@ -9,50 +9,69 @@
     <script src="table.js"></script>
     <script src="filters.js"></script>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="table-style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@1,300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
   </head>
     
-  <body onload="mainLoad()">
-    <h1>Main Page</h1>
-
+  <body onload="mainLoad(); createFilters()">
```

Comment:
Can we just call createFilters in mainLoad?

Reply:
Yes